### PR TITLE
Create DB Animal Table

### PR DESCRIPTION
**Pull request by HSMathebula**
This PR contains the implementation of the Vet Clinic database.

> Highlights:

- Created a database: `vet_clinic`
- Created a table named animals with the following columns:

```
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
```

- I copied the SQL used in the previous point into the schema.sql file.
- I Inserted the following data into the ANIMALS table:

```
Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
```

- Copy the SQL you used in the previous point into the data.sql file in the generated repository.
- Write queries for the following quests:

```
List the name of all animals born between 2016 and 2019.
List the name of all animals that are neutered and have less than 3 escape attempts.
List the date of birth of all animals named either "Agumon" or "Pikachu".
List name and escape attempts of animals that weigh more than 10.5kg
Find all animals that are neutered.
Find all animals not named Gabumon.
Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
```

- Here is the screenshot for the query results:

db11

db22

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+VALUES ('Agumon', date '2020-02-03', 0, 'true', 10.23);
+
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Gabumon', date '2018-11-15', 2, 'true', 8);
+
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Pikachu', date '2021-1-7', 1, 'false', 15.04);
+
+INSERT INTO animal (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES('Devimon', date '2017-05-12', 5, 'false', 11);
```
Comment (mahbubul14) [inline data.sql]: - [ ] Neutered should be true for the Devimon. Kindly fix it. You can also check the following example to get an idea:
  `EX: VALUES ('Devimon', '2017/05/12', '5', 'true', '11.0');`

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @HSMathebula ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ]  Currently, your `README` file is not updated according to the project. Please update your `README` file. You can use [this template ](https://github.com/microverseinc/readme-template)for the `README` but remember to customize it to your project.

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by HSMathebula** [comment]
@mahbubul14 I have fixed the issues thank

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @HSMathebula, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- You have done a great job working on the requested changes ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**